assets/src/components/activities/common/choices/authoring/ChoicesAuthoring.tsx

```diff
@@ -61,15 +67,23 @@ export const Choices: React.FC<Props> = ({
                     onChange={(e) => onEdit(choice.id, makeContent(e.target.value).content)}
                   />
                 ) : (
-                  <RichTextEditorConnected
+                  <SlateOrMarkdownEditor
```

Comment:
This is a common pattern you'll see in a lot of places... Replace the RichText editor with this new editor that can switch back and forth, and set up an appropriate onEditorTypeChange handler.